src/views/Main/Container.js

```diff
@@ -16,7 +16,7 @@ export class Container extends React.Component {
 
   constructor() {
     super();
-    Loom.configure({ baseUrl: 'localhost' });
+    Loom.configure({ baseUrl: 'api.loom.digital' });
```

Comment:
it would be better if this happens automatically based on the environment the app is running in, i.e. local, staging, prod